app/models/minister-position-function.js

```diff
@@ -0,0 +1,5 @@
+import Model, { attr } from '@ember-data/model';
+
+export default class MinisterPositionFunctionModel extends Model {
+  @attr('string') label;
```

Comment:
Transforms should only be used if the backend sends the wrong type. In this case a string is already sent so we can drop the transform.
```suggestion
  @attr label;
```